fieldservice_project/models/project_task.py

```diff
@@ -0,0 +1,31 @@
+# Copyright (C) 2019 - TODAY, Patrick Wilson
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models, _
```

Comment:
Travis error [https://travis-ci.org/OCA/field-service/jobs/576997243#L1167-L1170](url)

```suggestion
from odoo import api, fields, models
```

Reply:
Fixed